packages/js/product-editor/src/blocks/generic/linked-product-list/edit.tsx

```diff
@@ -137,13 +140,17 @@ export function LinkedProductListBlockEdit( {
 			fallbackToRandomProducts: true,
 		} ) ) as Product[];
 
+		setIsChoosingProducts( true );
```

Comment:
Shouldn't this go above the `getRelatedProducts` as I am not seeing the `isBusy` state, I think its going to fast.

Reply:
oh right. I introduced this mistake when rebasing the PR.

Reply:
addressed here https://github.com/woocommerce/woocommerce/pull/43656/commits/a3f1c018326ef69aec914560a25b6a8ec5891092